app/models/video.py

```diff
@@ -1,4 +1,21 @@
 from app import db
 
 class Video(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    #specifying a tablename that is plural to match everywhere else.
+    #__tablename__ = "Videos"
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime)
+    #available_inventory should be an attribute of the video
+    available_inventory = db.Column(db.Integer)
```

Comment:
I just want to point out that the available inventory can be calculated by taking the total inventory and subtracting the number of rentals that are still outstanding.